examples/chat/client/chat.cc

```diff
@@ -251,7 +251,9 @@ int main(int argc, char** argv) {
     LOG(QFATAL) << "Failed to create application stub";
   }
 
-  if (!absl::GetFlag(FLAGS_test)) {
+  if (absl::GetFlag(FLAGS_test)) {
+    return EXIT_SUCCESS;
+  } else {
```

Comment:
I think you can remove the else, an early return is clear enough, and you can remove a layer of indentation, but up to you.

Reply:
Done